mapzen-places-api/src/main/java/com/mapzen/places/api/internal/AutocompleteDetailFetchListener.java

```diff
@@ -0,0 +1,39 @@
+package com.mapzen.places.api.internal;
+
+import com.mapzen.android.lost.api.Status;
+import com.mapzen.android.lost.internal.DialogDisplayer;
+import com.mapzen.android.lost.internal.SettingsDialogDisplayer;
+import com.mapzen.places.api.Place;
+
+/**
+ * Handles notifying the {@link PlaceAutocompleteController} after a place's details have either
+ * been successfully retrieved or failed to be retrieved.
+ */
+class AutocompleteDetailFetchListener implements OnPlaceDetailsFetchedListener {
+
+  final PlaceAutocompleteController controller;
+  final DialogDisplayer dialogDisplayer = new SettingsDialogDisplayer();
```

Comment:
Same comment as `AutocompletePendingResult` can we just use `null` instead of a real `DialogDisplayer` until we come up with a better way to scope this class to the Lost `SettingsApi`?